Updated the applying knowledge graph section. (#20)
greenelab/knowledge-graph-review: content/04.network_applications.md

## Applying Knowledge Graphs to Biomedical Challenges

> Knowledge graphs can help researchers tackle many important biomedical tasks such as finding new treatments for FDA approved drugs [@doi:10.7554/eLife.26726], performing patient diagnoses [@arxiv:1611.07012] and predicting associations between diseases and RNA [@doi:10.1155/2017/2498957].

Reviewer: ```suggestion
Knowledge graphs can help researchers tackle many biomedical tasks such as finding new treatments for existing drugs [@doi:10.7554/eLife.26726], aiding efforts to diagnose patients [@arxiv:1611.07012] and predicting associations between diseases and biomolecules [@doi:10.1155/2017/2498957].
```

> These tasks are accomplished by representing knowledges graphs in a low dimensional space, which is a process called representation learning.

Reviewer: ```suggestion
In many cases, solutions rely on representing knowledges graphs in a low dimensional space, which is a process called representation learning.
```

This space preserves a knowledge graph's local and/or global structure and 

> is used with machine learning models to make predictions or infer new knowledge.

Reviewer: ```suggestion
This space preserves a knowledge graph's local and/or global structure and can support efforts to apply machine learning methods to make predictions.
```

> In this section, we discuss techniques that construct this low dimensional space (Unifying Techniques) and discuss applications that use this space to solve biomedical problems (Unifying Applications).

Reviewer: ```suggestion
We discuss the unifying techniques to construct this low dimensional space and unifying applications that use this space to solve biomedical problems.
```
